index.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" >
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>My Portfolio </title>
+   <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.8.2/css/all.css">
+   <link href='https://fonts.googleapis.com/css?family=Poppins' rel='stylesheet' type="text/css">
+    <link rel="stylesheet" href="style.css">
+  </head>
+      <body>
+      <div class="flex-container">
+        <div class="item1">
+          <header class="header">
+            <nav>
+           
+            <button class="button"><a href="#">Logo</a></button> 
+            <a href="#"><i class="fa fa-bars"></i></a>
+            </nav>
+         </header>
+        </div>
+        <div class="item2">
+          <section>
+            <h1 class="mariane">I'm Mariane<br>
+              Glad to see<br>
+             you!<br>
+           </h1>
+          </section>
+            <section>
+            <p class="text">
+              I’m a software developer! I can help you<br>
+               build a product , feature or website Look <br>
+               through some of my work and<br>
+                experience! If you like what you see and <br>
+                have a project you need coded, don’t<br>
+                 hestiate to contact me.
+            </p>
+          </section>
+          <section >
+            <h2 class="cap">LET’S CONNECT</h2>
+          </section>
+          
+<section class="img">
+  <ul>
+    <li><a href="#"><i class="fab fa-twitter"></i></a></li>
+    <li><a href="#"><i class='fab fa-linkedin-in'></i></a></li>
+    <li><a href="#"><i class='fab fa-medium-m'></i></a></li>
+    <li><a href="#"><i class='fab fa-github'></i></a></li> 
+    <li><a href="#"><i class='fab fa-angellist'></i></a></li> 
```

Comment:
- [ ] As requested by the previous reviewer, kindly make these icons look more like the ones in Figma, this includes the color, the spacing between them and the alignment of the list 👍 

## Your design:
![image](https://user-images.githubusercontent.com/50721479/198334345-b3874737-7c5a-4e74-9015-e55374a914ee.png)

## Figma design:
![image](https://user-images.githubusercontent.com/50721479/198334395-3d8b3025-8af5-4dcc-a2cc-d1cc219f479d.png)